core/src/main/java/nl/nn/adapterframework/doc/InfoBuilderSource.java

```diff
@@ -122,7 +122,12 @@
 	static {
 		// Exclude classes that will give conflicts with existing, non-compatible bean definition of same name and class
 		excludeFilters.add("nl\\.nn\\.adapterframework\\.extensions\\.esb\\.WsdlGeneratorPipe");
+		excludeFilters.add("nl\\.nn\\.adapterframework\\.extensions\\.ifsa\\.jms\\.IfsaFacade");
+		excludeFilters.add("nl\\.nn\\.adapterframework\\.extensions\\.ifsa\\.jms\\.IfsaRequesterSender");
 		excludeFilters.add("nl\\.nn\\.adapterframework\\.extensions\\.ifsa\\.IfsaRequesterSender");
+		excludeFilters.add("nl\\.nn\\.adapterframework\\.extensions\\.ifsa\\.jms\\.PullingIfsaProviderListener");
+		excludeFilters.add("nl\\.nn\\.adapterframework\\.extensions\\.ifsa\\.jms\\.PushingIfsaProviderListener");
+		excludeFilters.add("nl\\.nn\\.adapterframework\\.extensions\\.ifsa\\.jms.IfsaProviderListener");
```

Comment:
Ik denk niet dat het nodig is om deze ifsa classes expliciet te excluden. Het is niet erg als ze niet gevonden worden, maar helemaal onderdrukken hoeft ook niet.